pages/common/phpstan.md

```diff
@@ -9,9 +9,9 @@
 
 - Analyze a directory using a configuration file:
 
-`phpstan analyse {{path/to/directory}} --configuration {{path/to/config}}`
+`phpstan analyse {{path/to/directory}} {{[-c|--configuration]}} {{path/to/config}}`
 
-- Analyze using a specific rule level (0-7, higher is stricter):
+- Analyze using a specific rule level (0-10, higher is stricter):
 
 `phpstan analyse {{path/to/directory}} --level {{level}}`
```

Comment:
```suggestion
`phpstan analyse {{path/to/directory}} {{[-l|--level]}} {{level}}`
```
You can add the shortform options all of the options. The github UI doesn't allow for making suggestions past this point.